src/weather_model_graphs/create/archetype.py

```diff
@@ -18,20 +23,24 @@ def create_keisler_graph(xy_grid, grid_refinement_factor=3):
 
     Parameters
     ----------
-    xy_grid: np.ndarray
-        2D array of grid point positions.
-    grid_refinement_factor: float
-        Refinement factor between grid points and mesh
+    coords: np.ndarray
+        2D array of grid point positions, either in-projection Cartesian coordinates or lat-lons
+    mesh_node_distance: float
+        Distance (in x- and y-direction) between created mesh nodes,
+        in coordinate system of coords
+    projection: cartopy.crs.CRS or None
```

Comment:
I think this docstring line needs updating to reflect that `coords_crs` and `graph_crs` are now provided separately.

Reply:
Right, forgot to change those docstrings as well. Thanks!